### PR TITLE
Preprocess Windows encoding

### LINKED ISSUE #187: Repl displays extra symbols
I use `lldb-vscode` from LLVM and it displays ending symbols at the end of newline:
изображение

Is it possible to hide it?

### PR DESCRIPTION
**Pull request by Shatur**
Was suggested by @theHamsta. Closes #187.

### CODE REVIEW

File: lua/dap/repl.lua [hunk 1]
```diff
@@ -270,6 +270,9 @@ function M.append(line, lnum)
     if api.nvim_get_current_win() == repl.win and lnum == '$' then
       lnum = nil
     end
+    if api.nvim_buf_get_option(repl.buf, 'fileformat') ~= 'dos' then
+      line = line:gsub('\r\n', '\n')
```
Comment (theHamsta) [inline lua/dap/repl.lua:274]: maybe `api.nvim_buf_get_option(repl.buf, 'fileformat')  '\n' or '\r'`?

Comment (Shatur95) [reply lua/dap/repl.lua:274]: Do you mean to use a ternary expression for splitting?

Comment (mfussenegger) [reply lua/dap/repl.lua:274]: Both variants seem okay to me. Let me know how you decide and I'll merge

Comment (Shatur) [reply lua/dap/repl.lua:274]: I would leave the current change because I don't understand what @theHamsta means.

Comment (theHamsta) [reply lua/dap/repl.lua:274]: I meant to convert to \n only on Unix, on Mac we need \r to have consistent line endings

Comment (Shatur) [reply lua/dap/repl.lua:274]: Makes sense! @mfussenegger, will you push the changes?

**Pull request merged**